config/feature-flags.yml

```diff
@@ -0,0 +1,4 @@
+# feature_flag_test:
```

Comment:
ohh why did you remove the "schema" definition? It was cool!

Reply:
@santib not sure why 😃 , I've added it back again